dirs/dirs.go

```diff
@@ -106,6 +106,7 @@ var (
 	SnapshotsDir     string
 
 	ErrtrackerDbDir string
+	SysDir          string
```

Comment:
Nitpick, to avoid any shade of confusion could you please call this `SysfsDir` instead?